src/braket/ocean_plugin/braket_dwave_sampler.py

```diff
@@ -34,22 +34,22 @@ class BraketDWaveSampler(BraketSampler):
     A class for using DWave-formatted parameters and properties with Amazon Braket as a sampler.
 
     Args:
-        s3_destination_folder (AwsSession.S3DestinationFolder): NamedTuple with bucket (index 0)
-            and key (index 1) that is the results destination folder in S3.
+        s3_destination_folder (AwsSession.S3DestinationFolder, optional): NamedTuple with bucket
+        (index 0) and key (index 1) to save the task's results in S3.
+        Default is `<default_bucket>/tasks`.
         device_arn (str): AWS quantum device arn. Default is the latest D-Wave QPU device ARN.
-        aws_session (AwsSession): AwsSession to call AWS with.
+        aws_session (AwsSession): AwsSession to call AWS with. Default is `None`.
```

Comment:
Default generally refers to the logical default used by the function, rather than the fallback parameter in the function signature. For example, see [quantum job creation](https://github.com/aws/amazon-braket-sdk-python/blob/main/src/braket/aws/aws_quantum_job.py#L161) for a similar case. I believe the default here should be `AwsSession()`

Reply:
Also, we're pretty inconsistent with when we list "optional" in the docs throughout the service, but I think it would be best to be consistent with treatment of listing "optional" and defaults within the same doc string